### ISSUE TITLE
Alerts set up with Microsoft Teams is failing to send notification 

### ISSUE THREAD

**Issue by kerenkhatiwada** [getsentry/sentry#69147]
### Environment

SaaS (https://sentry.io/)

### Steps to Reproduce

Notifications are failing to be sent to MS Teams. When sending a test notifications from the alerts page using the “Send Test Notification” fails with "Internal Error".
Here is an example response when attempting to send a test notification:
{"detail":"Internal Error","errorId":"3a2b5a3da47a4172895655d4c8bacb97"} . [Here](https://sentry.sentry.io/issues/4984058313/?project=1&query=id%3A3a2b5a3da47a4172895655d4c8bacb97&referrer=issue-stream&statsPeriod=7d&stream_index=0) is the corresponding error. 

### Expected Result

Notifications are sent to MS teams when the alert is triggered and  the “Send Test Notification” option results in a notification in MS teams

### Actual Result

Notifications are also failing to be sent

### Product Area

Settings - Integrations

**Comment by rowanmoul** [comment]
I am the one who first reported this issue. Please let me know if I can help with any further details!

**Comment by Dhrumil-Sentry** [comment]
We've added this to our backlog and we will comment here once we have more details

**Comment by rowanmoul** [comment]
I re-installed the Teams "app" today at @kerenkhatiwada's suggestion. We had already done this a few days ago before contacting support and it didn't help. Today when I did it, it appears to mostly be working, with the exception on *one* of our channels. I can't figure out what is different about this one channel, but for now I have created a new channel as a workaround, and test alerts are able to be posted there.
